frontend/vue/components/CodeExercise/CodeOutput.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div ref="outputDiv" class="code-output">
+    <div class="code-output__state-info__wrapper">
+      <div v-if="!kernel && !error" class="code-output__state-info">
+        <bx-loading class="code-output__state-info__icon" assistive-text="Connecting to the server" type="small" />
+        Connecting to the server
```

Comment:
Similar comment regarding translations, something like:

```
{{ $translate('Connecting to the server') }}
```

then make sure the strings are defined in `strings.yaml` file